.github/workflows/upload.yml

```diff
@@ -83,26 +83,11 @@ jobs:
            ./middleware
          outPath: pwa.tar.gz
 
-      - name: Upload Artifact
```

Comment:
It works, but I think we should rename the job from "install-and-build-dependencies" to something else

maybe: build-and-upload

Reply:
happy? 👯